_layouts/topic.html

```diff
@@ -59,7 +59,12 @@
     {% else %}
       {% assign bold='' %}
     {% endif %}
-    {% capture references %}{{references}}{{newline}}- {{bold}}{{mention.date}} [{{mention.title}}]({{mention.url}}){{bold}}{% endcapture %}
+    {% if mention.url contains "/en/newsletters" %}
+      {% assign date = mention.url | remove_first: "/en/newsletters/" | slice: 0, 10 | replace: "/", "-" %}
+    {% else %}
+      {% assign date = mention.date %}
```

Comment:
If we make `date:` non-mandatory in the YAML, we need to catch the case here of entries that aren't newsletters but also don't have dates defined.  E.g. (untested):

```
[...]
{% else %}
  {% if mention.date == empty %}
    {% include ERROR_44_MISSING_DATE %}
  {% else %}
    {% assign date = mention.date %}
  {% endif %}
{% endif %}
[...]
```